cmd/cli/plugin/package/kctrl/README.md

```diff
@@ -523,6 +549,114 @@ Use "tanzu package repository [command] --help" for more information about a com
     11:47:38PM: packagerepository/standard-repo (packaging.carvel.dev/v1alpha1) namespace: test-ns: DeletionSucceeded
     ```
 
+15. Create a package
+
+    ```sh
+    >>> tanzu package init
```

Comment:
Question: Is the command ```tanzu package init``` or ```package init```?

Reply:
Command is `package init` only. `tanzu` is the "binary name" here, it could be `kctrl` as well, that's why in the informational text, we are just using the command and not including the binary name.